test/tests/functional/pbs_validate_job_qsub_attributes.py

```diff
@@ -133,7 +133,11 @@ def test_qsub_N_job_array(self):
         This test case validates, illlegal characters in job name
         for a job array, cause qsub to error out
         """
-        J = Job(TEST_USER, attrs={ATTR_N: 'j&whoami&g;/tmp/b&', ATTR_J: '1-2'})
+        J = Job(
```

Comment:
why did you change the formatting of this line? It looks much harder to read now. I suggest reverting it to how it was before.